PROJECT

```diff
@@ -7,12 +7,12 @@ projectName: selfserviceoverlay
 repo: github.com/AlonaKaplan/selfserviceoverlay
 resources:
 - api:
-    crdVersion: v1
+    crdVersion: v1alpha1
```

Comment:
Some more places I found v1 version that may need to be changed as well.

config/manager/manager.yaml:apiVersion: v1
config/rbac/service_account.yaml:apiVersion: v1
config/samples/kustomization.yaml:- self.service_v1_overlaynetwork.yaml
config/samples/self.service_v1_overlaynetwork.yaml:apiVersion: self.service.ovn.org/v1

Reply:
Who generated those?

Reply:
I think it was part of the project initialization. Now it has to be updated manually.